src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsexternalusersapi/jpa/repository/RoleRepository.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.hmppsexternalusersapi.jpa.repository
+
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.CrudRepository
+import org.springframework.data.repository.query.Param
+import org.springframework.lang.NonNull
+import uk.gov.justice.digital.hmpps.hmppsexternalusersapi.model.Authority
+
+interface RoleRepository : CrudRepository<Authority, String>, JpaSpecificationExecutor<Authority> {
+  @NonNull
+  @Query("Select * from Roles r where r.admin_type LIKE %:adminType% order by r.role_name", nativeQuery = true)
+  fun findAllByOrderByRoleNameLike(@Param("adminType") adminType: String): List<Authority>
```

Comment:
I have copied this from hmpps-auth.  Because this is now in external-users should AdminType never be a param, and we just pass through the "EXT_ADM" one?